tools/lc

```diff
@@ -25,23 +25,23 @@ function cmd_project_new {
   fi
 
   echo "—> creating files hierarchy"
-  mkdir -p $project_name/{build,idl,inc/database,src/database}
+  mkdir -p $project_name/{build,idl,inc/$coin_name/database,src/$coin_name/database}
   touch $project_name/README.md
 
   echo "—> symlinking ledger-core’s libraries"
   ln -sf $PWD/ledger-core/lib $project_name/lib
   ln -sf $PWD/ledger-core/cmake $project_name/cmake
 
   echo "—> generating CMake configuration"
-  sed s/\$project_name/$project_name/g ./tools/templates/CMakeLists-project.txt.tpl > $project_name/CMakeLists.txt
-  sed s/\$project_name/$project_name/g ./tools/templates/CMakeLists-project-src.txt.tpl > $project_name/src/CMakeLists.txt
+  sed -e s/\$project_name/$project_name/g ./tools/templates/CMakeLists-project.txt.tpl > $project_name/CMakeLists.txt
+  sed -e s/\$project_name/$project_name/g -e s/\$coin_name/$coin_name/g ./tools/templates/CMakeLists-project-src.txt.tpl > $project_name/src/CMakeLists.txt
```

Comment:
This should have been in a different PR, because now I’m going to have issues with XRP.